portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,65 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="home">
+        <h1>Jimin Byun</h1>
```

Comment:
nit: please use 2 spaces for indentation: https://google.github.io/styleguide/htmlcssguide.html#Indentation

Reply:
Done!